package.json

```
@@ -29,6 +29,7 @@
     "android-build": "fastlane android build",
     "android-build-e2e": "bundle exec fastlane android build_e2e",
     "test": "TZ=utc jest",
+    "typecheck": "tsc --project tsconfig.json",
```

NAB but is this necessary? Seems like this could just be:

```suggestion
    "typecheck": "tsc",
```

at which point, having the script in package.json isn't as worth it imo

---

1. That's right, `--project tsconfig.json` isn't necessary. I'll remove it! 
2. I believe it's nice to have Typescript script defined in package.json even when it is short. This way it is clear what is used for type checking. It also makes sure that everybody uses the same `tsc` version, here is an example:
```
 ~/Documents/swmansion/expensify/App tsc -v
Version 4.6.4
 ~/Documents/swmansion/expensify/App npm run typecheck

> new.expensify@1.3.26-1 typecheck
> tsc -v

Version 4.8.4
```